### ISSUE TITLE
Button for suspend/resume cronjob on detail page

### ISSUE THREAD

**Issue by dzoeteman** [kubernetes/dashboard#2836]
##### Environment

```
Dashboard version: master
Kubernetes version: 1.8.5
```
It would be nice to have an easy "suspend" and "resume" button on the cronjob detail page, to easily suspend cronjobs. Right now you need to actually edit the json.

This is useful when running in production and you want to stop cronjobs for reasons like debugging etc.

**Comment by vaibhavsingh97** [comment]
@floreks Can I take this issue? Any resources to fix this issue would be a great help

**Comment by floreks** [comment]
We are migrating frontend to angular 5, so we'd have to rewrite it anyway. I'd recommend you to wait until we finish and start with beta tests. You can then work on it with new angular.

**Comment by rahulb3521** [comment]
Hi @floreks I would also like to contribute to this ticket let me know once the migration is done. I can collaborate with @vaibhavsingh97 and work together on this. I have a strong grip on angular/react/vue and other UI frameworks.

**Comment by maciaszczykm** [comment]
@rahulb3521 It is done already. Check master.

**Comment by yinyangsx** [comment]
Hi, this function is needed for me, whether this function has been implemented?

**Comment by floreks** [comment]
There is only a trigger option to create a job based on the cron job. There is no start/stop option.

**Comment by Akash3121** [comment]
Hi @maciaszczykm,
Is this issue fixed?

**Comment by maciaszczykm** [comment]
@Akash3121 No, it is not done yet. In my previous comment I was referring to migration to the new Angular.

**Comment by mtbuzato** [comment]
Hi, I would like to work on this if if not resolved yet @maciaszczykm

**Comment by sswastik02** [comment]
Hi @maciaszczykm ,
I see this issue is not yet resolved. If it is still valid, I would like to work on it.
